@hans2103 Please fix the javascript cs issues reported here https://ci.joomla.org/joomla/joomla-cms/77860/1/33 :

```
/drone/src/build/media_source/templates/administrator/atum/js/template.es6.js
  251:25  error  Expected parentheses around arrow function argument  arrow-parens
  252:36  error  Unexpected unnamed function                          func-names
  252:36  error  Unexpected function expression                       prefer-arrow-callback
  253:29  error  Expected parentheses around arrow function argument  arrow-parens
  256:22  error  Missing trailing comma                               comma-dangle
```

--- a/build/media_source/templates/administrator/atum/js/template.es6.js
+++ b/build/media_source/templates/administrator/atum/js/template.es6.js
@@ -246,7 +246,7 @@ window.addEventListener('joomla:menu-toggle', (event) => {
 /**
  * Close any open data-bs-toggle="collapse" when opening a data-bs-toggle="dropdown"
  *
- * since 4.4
+ * @since 4.4
  */
 document.querySelectorAll('[data-bs-toggle="dropdown"]').forEach((button) => {
   button.addEventListener('click', () => {

--- a/build/media_source/templates/administrator/atum/js/template.es6.js
+++ b/build/media_source/templates/administrator/atum/js/template.es6.js
@@ -243,17 +243,17 @@ window.addEventListener('joomla:menu-toggle', (event) => {
   }
 });
 
-// Get all buttons with data-bs-toggle attributes
-const collapseButtons = document.querySelectorAll('[data-bs-toggle="collapse"]');
-const dropdownButtons = document.querySelectorAll('[data-bs-toggle="dropdown"]');
-
-// Add event listeners to dropdown buttons
-dropdownButtons.forEach(button => {
-  button.addEventListener('click', function () {
-    collapseButtons.forEach(cb => {
+/**
+ * Close any open data-bs-toggle="collapse" when opening a data-bs-toggle="dropdown"
+ *
+ * since 4.4
+ */
+document.querySelectorAll('[data-bs-toggle="dropdown"]').forEach((button) => {
+  button.addEventListener('click', () => {
+    document.querySelectorAll('[data-bs-toggle="collapse"]').forEach((cb) => {
       const target = document.querySelector(cb.getAttribute('data-bs-target'));
       const collapseMenu = bootstrap.Collapse.getInstance(target) || new bootstrap.Collapse(target, {
-        toggle: false
+        toggle: false,
       });
       collapseMenu.hide();
     });

